monkey/common/types/networking.py

```diff
@@ -32,8 +32,10 @@ class NetworkService(Enum):
     """
 
     HTTP = "http"
+    HTTPS = "https"
```

Comment:
Wait, why is this not `auto()`, but a "member's name in all lower-case characters"?

Reply:
As far as I know, we need the names of the service to be displayed in the report.